src/Components/test/testassets/TestServer/Startup.cs

```diff
@@ -68,9 +69,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 </html>");
                 var content = writer.ToString();
                 ctx.Response.ContentLength = content.Length;
-                using var responseWriter = new StreamWriter(ctx.Response.Body);
```

Comment:
`StreamWriter.FlushAsync` does a synchronous write. I noticed this when I ran the app and Kestrel complained about this. Noticed this when I was debugging this failing blazor test.

Reply:
Flush or Dispose? `await using var...`?

Reply:
Why did this ever use StreamWriter. That's just a waste 😄

Reply:
Not using `StreamWriter` is a waste of an opportunity to use `StreamWriter`.